dsmr_parser/clients/serial_.py

```diff
@@ -84,16 +95,21 @@ async def read(self, queue):
             # Read line if available or give control back to loop until new
             # data has arrived.
             data = await reader.readline()
-            self.telegram_buffer.append(data.decode('ascii'))
+            try:
+                decoded_data = data.decode('ascii')
+            except:
+                pass
```

Comment:
Maybe put a logger statement here:
`logger.warning('Failed to decode telegram data: %s', e)`